projects/components/src/utils/unit/unit.ts

```diff
@@ -102,7 +102,27 @@ export abstract class ThousandsUnit extends Unit {
         }
     }
 
-    abstract getAllUnitTypes(): ThousandsUnit[];
+    abstract getAllUnitTypes(): NumberUnit[];
+
+    protected abstract getComparisonNumber(): number;
```

Comment:
Can we rename this to `getGroupingNumber()` and add some comments.

Reply:
`getGroupingNumber` is not clear to me.
Isn't it related to a `multiplier` ?

Reply:
Yea, I think Ivo's right. How does `getUnitMultiplier` sound?